tools/releaseBuild/azureDevOps/diagram.puml

```diff
@@ -21,14 +21,14 @@ folder "Windows Builds" as WinBuilds {
 
 agent "ComponentRegistration"  as BuildCG
 
-folder "Linux Package Scanning and Upload" {
+folder "Linux Package Scanning and Upload" as PkgScanUploadLinux {
```

Comment:
Scanning or signing?

Reply:
Scanning.  Linux signing has not been automated by the lab.